learning_journal_basic/templates/detail.jinja2

```diff
@@ -1,9 +1,9 @@
 {% extends 'index.jinja2' %}
 {% block body %}
-  <h1>{{ title }}</h1>
+  <h1>{{ article.title }}</h1>
   <hr />
-  <p> Created <strong>{{ creation_date }}</strong></p>
+  <p> Created <strong>{{ article.date_created }}</strong></p>
   <hr />
-  <p>{{ body }}</p>
-  <a class="icon-books nav-item-1" data-tabs="edit_article" href="{{ request.route_url('edit_view', id=id) }}"></a><a class="nav-item-2" data-tabs="educationTab" href="{{ request.route_url('edit_view', id=id) }}">Edit</a>
+  <p>{{ article.body }}</p>
+  <a class="icon-books nav-item-1" data-tabs="edit_article" href="{{ request.route_url('edit_view', id=article.id) }}"></a><a class="nav-item-2" data-tabs="educationTab" href="{{ request.route_url('edit_view', id=article.id) }}">Edit</a>
```

Comment:
When rendered this button is not obviously clickable or a link. You might want to make this into something more visually obvious.